### PR TITLE
Added two test cases validating unzipped folder

### PR DESCRIPTION
**Pull request by Mariusz-Poznanski-Mobica**

### CODE REVIEW

**Review by piotr-markiewicz-mobica — APPROVED** [review]

File: tests/src/page_objects/keywords/common.robot [hunk 1]
```diff
@@ -98,5 +99,39 @@ Unzip File
...
+    ${downloaded_file_path_new_name}    Replace String    ${downloaded_file_path}    ${space}    ${empty}
+    Copy Directory   ${downloaded_file_path}    ${downloaded_file_path_new_name}
+    Wait Until Created    ${downloaded_file_path_new_name}
+    Log To Console    Removed spaces from folder name: ${downloaded_file_path_new_name}
+    RETURN    ${downloaded_file_path_new_name}
+
+Compare Directory And Files Tree
+    [Arguments]    ${path_to_unzipped_folder}    ${path_to_unzipped_folder_pattern}
+    ${rc}    ${output}    Run And Return Rc And Output   diff -qr ${path_to_unzipped_folder} ${path_to_unzipped_folder_pattern}
+    Log To Console    rc: ${rc}
```
Comment (krzysztof-rodzaj-mobica) [inline tests/src/page_objects/keywords/common.robot]: I wonder if these logs are needed - they do not show in the log.html and it is checked two lines after anyway

File: tests/tests/large_files_tests.robot [hunk 2]
```diff
@@ -16,31 +16,81 @@ Test Teardown    Clean After Test
...
     Wait Until Created    ${downloaded_file_path_new_name}
-
+    # Unzip extracted ZIP file
     Validate and Unzip Test File    ${downloaded_file_path_new_name}
-    Check If Log Console Contains    "status":"Completed successfully"
-    Check If Log Console Does Not Contain Errors
-    Check If JS Console Does Not Contain Errors
\ No newline at end of file
+    ${unzipped_new_folder_name}=     Remove Spaces From Directory Name   ${DOWNLOAD_PATH}${TEST_FILE}[archive_name]
+    Log To Console    unzipped new folder name: ${unzipped_new_folder_name}
```
Comment (krzysztof-rodzaj-mobica) [inline tests/tests/large_files_tests.robot]: I wonder if this log to console is needed

File: tests/src/page_objects/keywords/common.robot [hunk 3]
```diff
@@ -98,5 +99,39 @@ Unzip File
...
+    Log To Console    output: ${output}
+    Should Be Equal As Integers    ${rc}    0
+    Log To Console    Directory tree is the same as pattern: OK.
+
+ Compare Cheksum For Each File
+    [Arguments]    ${path_to_unzipped_folder}    ${path_to_unzipped_folder_pattern}
+    Log To Console    Checking checksums for    ${path_to_unzipped_folder}
+    # rclone is a programm comparing file sizes and hashes for each file in a given path
+    ${rc}    ${output}    Run And Return Rc And Output    rclone check ${path_to_unzipped_folder_pattern} ${path_to_unzipped_folder}
+    Log To Console    rc: ${rc}
```
Comment (krzysztof-rodzaj-mobica) [inline tests/src/page_objects/keywords/common.robot]: I wonder if these logs are needed - they do not show in the log.html and it is checked two lines after anyway

File: tests/src/page_objects/keywords/common.robot [hunk 4]
```diff
@@ -98,5 +99,39 @@ Unzip File
...
+    Log To Console    otuput: ${output}
```
Comment (krzysztof-rodzaj-mobica) [inline tests/src/page_objects/keywords/common.robot]: I wonder if we couldn't check the output for errors in the next line

mistake in "output" word

File: tests/tests/large_files_tests.robot [hunk 5]
```diff
@@ -16,31 +16,81 @@ Test Teardown    Clean After Test
 *** Variables ***
 ${TEST_FILE}           ${file_1}
 ${EXTRACTION_TIMEOUT}    ${file_1}[extraction_time]
+${PATH_TO_PATERRN_FOLDER}    ${file_1}[pattern_path]
```
Comment (krzysztof-rodzaj-mobica) [inline tests/tests/large_files_tests.robot]: Mistake in "pattern" word in variable name

File: tests/src/page_objects/keywords/common.robot [hunk 6]
```diff
@@ -98,5 +99,39 @@ Unzip File
...
+    Copy Directory   ${downloaded_file_path}    ${downloaded_file_path_new_name}
+    Wait Until Created    ${downloaded_file_path_new_name}
+    Log To Console    Removed spaces from folder name: ${downloaded_file_path_new_name}
+    RETURN    ${downloaded_file_path_new_name}
+
+Compare Directory And Files Tree
+    [Arguments]    ${path_to_unzipped_folder}    ${path_to_unzipped_folder_pattern}
+    ${rc}    ${output}    Run And Return Rc And Output   diff -qr ${path_to_unzipped_folder} ${path_to_unzipped_folder_pattern}
+    Log To Console    rc: ${rc}
+    Log To Console    output: ${output}
```
Comment (krzysztof-rodzaj-mobica) [inline tests/src/page_objects/keywords/common.robot]: I wonder if we couldn't check the output for errors in the next line

Comment (Mariusz-Poznanski-Mobica) [reply tests/src/page_objects/keywords/common.robot]: We check RC, if different than 0 then test must fail. I added output for debugging purposes.

**Review by krzysztof-rodzaj-mobica — APPROVED** [review]

**Pull request merged**